apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/README.md

```diff
@@ -0,0 +1,38 @@
+# Apiary Kafka Metastore Listener
+
+## Overview
+
+This module contains classes for publishing Hive Metastore events to Kafka.
+
+### Kafka Metastore Listener Configuration
+
+To configure the listener, the following Hadoop configuration can be used:
+
+| Parameter | Required | Default
+|:----|:----:|:----:|
+| `topic.name`   | Yes | n/a
+| `bootstrap.servers` | Yes | n/a
+| `client.id` | Yes | n/a
```

Comment:
This is optional now isn't it?

Reply:
No this is for the listener (producer). I've made this required as I've read that its recommended to set this option: https://docs.confluent.io/current/clients/producer.html